backend_simple.py

```
@@ -316,6 +414,117 @@ async def get_docs():
         "GET /api/v1/auth/me - Current user info"
     ]}
 
+@app.post("/api/v1/auth/logout")
+async def logout(current_user: Dict = Depends(get_current_user)):
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)